gems/http/5.1/http.rbs

```diff
@@ -0,0 +1,47 @@
+# Write the type definition here!
+
+module HTTP
+  interface _Stringable
+    def to_s: () -> String
+  end
+
+  interface _Callable
+    def call: () -> (Symbol | String)
+  end
+
+  def self.auth: (_Stringable) -> HTTP::Client
+
+  class Client
+    def initialize: (?Hash[untyped, untyped] default_options) -> void
+  end
+
+  class Request
+    attr_reader verb: _Stringable
+    attr_reader scheme: Symbol
+    attr_reader uri_normalizer: _Callable
+    attr_reader uri: HTTP::URI | _Stringable
+    attr_reader proxy: Hash[untyped, untyped]
+    attr_reader body: String | Enumerable[untyped] | IO | nil
+    attr_reader version: String
+
+    def initialize: (Hash[Symbol, untyped] opts) -> void
+  end
+
+  class Response
+    attr_reader status: Integer
+    attr_reader version: String
+    attr_reader body: String
+    attr_reader request: HTTP::Request
+    attr_reader proxy_headers: Hash[untyped, untyped]
+
+    def initialize: (Hash[Symbol, untyped] opts) -> void
+
+    # defined by def_delegator
```

Comment:
[suggest] This internal comment is shown as a public doc on IDE, so it seems better to remove it.

```suggestion
```